### PR TITLE
Notification ui support

### PR DESCRIPTION
**Pull request by abemac**

### CODE REVIEW

**Comment by abemac** [comment]
I think the checks should pass once https://github.com/Asymmetrik/node-rest-starter/pull/22 is merged

**Comment by brianghig** [comment]
> I think the checks should pass once [Asymmetrik/node-rest-starter#22](https://github.com/Asymmetrik/node-rest-starter/pull/22) is merged

@abemac the two repositories are separate, and we'd like to make sure that each can stand on their own moving forward (albeit with well-documented interfaces to help them connect to each other when needed).

Do you have insight into the travis build logs to see where this branch is failing?

**Comment by jrassa** [comment]
@abemac The errors were due to a lack of error handling on the http calls in `message.service.ts`.  I pushed an update to resolve this.

File: src/app/core/messages/view-all-messages/view-all-messages.component.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+$card-width: 800px;
+$card-icon-width: 31px;
+
+.card-message {
+	border: none;
+	-webkit-box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.298039215686275);
+	box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.298039215686275);
+	display: inline-block;
+	width: 100%;
+	max-width: $card-width;
```
Comment (abemac) [inline src/app/core/messages/view-all-messages/view-all-messages.component.scss:17]: should we add min-width too?

Comment (jrassa) [reply src/app/core/messages/view-all-messages/view-all-messages.component.scss:17]: I just added a min-width above the cards on `<section>`.  This ensure padding is maintained.

**Pull request merged**